misk-zookeeper/src/main/kotlin/misk/zookeeper/ZookeeperEnsembleProvider.kt

```diff
@@ -0,0 +1,48 @@
+package misk.zookeeper
+
+import org.apache.curator.ensemble.EnsembleProvider
+import org.slf4j.LoggerFactory
+
+/**
+ * ZookeeperEnsembleProvider is modeled after [FixedEnsembleProvider] but does not allow the
+ * connection string to be changed (a la [ExhibitorEnsembleProvider]). By default, Curator attempts
+ * to handle load balancing connections across ZooKeeper servers. However, the connection string
```

Comment:
thinking out loud, is this a misk layer concept or a skim one?

Reply:
Ah, yeah, this comment here is a bit Square specific, but the Provider itself could be useful to anyone who wants to handle load balancing themselves. I'll clean that up

Reply:
Eh, on second thought. Just going to move it back to skim